src/components/behandling/felles/utbetaling/OppdaterSimuleringKnapp.tsx

```diff
@@ -0,0 +1,27 @@
+import { Button } from '@navikt/ds-react';
+
+import style from './OppdaterSimulering.module.css';
+import { BehandlingId } from '~/types/BehandlingTypes';
+import { SakId } from '~/types/SakTypes';
+import { useOppdaterSimulering } from './useOppdaterSimulering';
+import { MeldekortBehandlingId } from '~/types/meldekort/MeldekortBehandling';
+
+type Props = {
+    sakId: SakId;
+    behandlingId: BehandlingId | MeldekortBehandlingId;
+};
+
+export const OppdaterSimuleringKnapp = ({ sakId, behandlingId }: Props) => {
+    const { oppdaterSimulering } = useOppdaterSimulering(sakId, behandlingId);
+    return (
+        <>
```

Comment:
Trenger ikke dette fragmentet kun for knappen. Men burde kanskje ha en alert ved errors her også?